components/k8s-model-server/iap-client/REAMDE.md

```diff
@@ -0,0 +1,60 @@
+# Serving with IAP enabled
+
+## Setup
```

Comment:
Lets put this in
https://github.com/kubeflow/kubeflow/tree/master/docs/gke

with other GKE specific docs.

Reply:
Done